app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt

```diff
@@ -225,6 +226,23 @@ class PatientEntryScreen(context: Context, attrs: AttributeSet) : RelativeLayout
     }
   }
 
+  private fun setInputFieldsHint() {
+    val allInputFields: Map<Class<*>, TextInputLayout> = mapOf(
+        PatientNameField::class.java to fullNameInputLayout,
+        LandlineOrMobileField::class.java to phoneNumberInputLayout,
+        BangladeshNationalIdField::class.java to bangladeshNationalIdInputLayout,
+        StreetAddressField::class.java to streetAddressInputLayout,
+        VillageOrColonyField::class.java to colonyOrVillageInputLayout,
+        ZoneField::class.java to zoneInputLayout,
+        DistrictField::class.java to districtInputLayout,
+        StateField::class.java to stateInputLayout
+    )
+
+    inputFields.inputFields.forEach {
```

Comment:
Seeing `inputFields.inputFields` feels weird. Can we rename one of the two? Maybe we can rename `InputFieds#inputFields` to just `fields`?

Reply:
Same comment for the edit patient screen.

Reply:
done